compiler/include/symbol.h

```diff
@@ -583,6 +589,9 @@ const char* intentDescrString(IntentTag intent);
 // pass-by-reference intents are used.
 bool argMustUseCPtr(Type* t);
 
+void substituteVarargTupleRefs(BlockStmt* ast, int numArgs, ArgSymbol* formal,
```

Comment:
This function is defined in resolution/functionResolution.cpp and so the prototype should be in
include/resolution.h.

When you move the signature please update the formatting to better handle the wrap.  As a general
rule, which is readily applied in this case, formals should either (1) fit on a single line or (2) be on
individual lines.